docs/diagrams/ListEvContactSequenceDiagram.puml

```diff
@@ -0,0 +1,77 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":ListEvContactCommandParser" as ListEvContactCommandParser LOGIC_COLOR
+participant "l:ListEvContactCommand" as ListEvContactCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "pr:EventSetContainsEventPredicate" as EventSetContainsEventPredicate MODEL_COLOR
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("listevcontact 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("listevcontact 1")
+activate AddressBookParser
+
+create ListEvContactCommandParser
+AddressBookParser -> ListEvContactCommandParser
+activate ListEvContactCommandParser
+
+ListEvContactCommandParser --> AddressBookParser
+deactivate ListEvContactCommandParser
+
+AddressBookParser -> ListEvContactCommandParser : parse("1")
+activate ListEvContactCommandParser
+
+create ListEvContactCommand
+ListEvContactCommandParser -> ListEvContactCommand
+activate ListEvContactCommand
+
+ListEvContactCommand --> ListEvContactCommandParser : l
```

Comment:
I think `: 1` can be excluded here

Reply:
Oh, this is supposed to be small letter L, which is the listevcontactcommand object, I changed it to ec for clearer understanding